presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,49 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
+* Fix accounting of raw input bytes in ScanFilterAndProjectOperator.
+* Fix an issue where server fails to start when two function namespace managers of the same type are specified.
+* Fix SQL function compilation error when input parameters contain lambda.
+* Fix ``IS DISTINCT FROM`` for long decimals.
+* Improve error handling for geometry deserialization edge cases.
+* Improve performance of array_join.
+* Improve efficiency of ``ST_Buffer``. The new implementation produces fewer buffer points on rounded corners, which will produce very similar but different results. It also better handles buffering with small (<1e-9) distances :func:`ST_Buffer`.
+* Improve OOM killer log output to put memory heavy nodes and queries first.
+* Improve efficiency by using JTS instead of Esri for many geometrical operations and fix incorrect calculation of extreme points in certain cases (:issue:`14031`).
+* Keep sum stat for string when min/max stat is too long in ORC writer.
```

Comment:
Is this an improvement or a fix? @wenleix

Reply:
I still don't understand this one.  what's this the "sum" of?  Is this for stats that go in the orc footer? Also, I think ORC writer things go under the Hive Changes.

Reply:
@rongrong : It's an improvement.

@rschlussel : It's the sum of length of the strings.  Previously, when the min/max of the string is too long, we drop all the stats on that column. so ORC Writer doesn't use too much memory. However, we could still keep the sum of length of that column. This can help, for example the IO team to do some estimation based on metadata :)

Reply:
ok moved it to HiveChanges @rschlussel  can you confirm if it looks good do we need to update the message

Reply:
Maybe also reword it to "Improve string stats by ..."?

Reply:
changed it to be "Improve string column stats by including the sum of lengths of the strings when individual min/max of the string is too long in ORC writer." @rongrong  does this look fine?